tests/fps_tests.py

```diff
@@ -5,7 +5,7 @@
 from unittest import mock
 from requests import structures
 
-sys.path.append('../first-party-sets')
+sys.path.append('tests/..')
```

Comment:
```suggestion
sys.path.append('.')
```

This is just a fancy way of saying "the current directory", right?

Reply:
I guess it is, I had been thinking that the interpreter might initially be reading from the tests directory. It seems silly that it's necessary to append the current directory so that the interpreter can read files from it. I would think that would be the default behavior.